README.rst

```diff
@@ -205,6 +205,18 @@ currently supported (these include ``customer.created``, ``customer.updated``,
 ``customer.subscription.created``, ``customer.subscription.deleted``
 ``invoice.created``, ``invoice.payment_succeeded``, ``invoice.payment_failed``).
 
+Flush stored data
+------------
```

Comment:
Again, this is invalid RST format.

Reply:
Sure I'll do it.
I was waiting for your review before doing anything on this pull request. ;)